0000-template.md

```diff
@@ -1,51 +1,48 @@
+- Title: (fill me in with a human-focussed title for the RFC)
 - Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
 - Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
-- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+- RFC PR: [#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Tracking Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+- Team: (fill me in with the team or teams responsible for this RFC)
```

Comment:
The label already covers this.

Reply:
It does, but part of the motivation for these changes is so that the author can suggest the team via tooling without needing to have that access to GH, and so that once committed, we still have a tool-visible record of information that was in the PR in the document itself.